google/cloud/redis/BUILD.bazel

```diff
@@ -31,11 +31,13 @@ filegroup(
 filegroup(
     name = "hdrs",
     srcs = glob([d + "*.h" for d in src_dirs]),
+    visibility = ["//:__pkg__"],
 )
```

Comment:
e.g.

```
filegroup(
    name = "public_hdrs",
    srcs = glob([d + "*.h" for d in service_dirs]),
    visibility = ["//:__pkg__"],
)
```

(Note the `service_dirs` instead of `src_dirs`)

Reply:
Good idea, that may be handy in a follow up PR.

Reply:
Included in this PR now.